### ISSUE TITLE
[erlang-ls/erlang-ls]: Erlang extension is missing compiled language server

### ISSUE THREAD

**Issue by bryphe** [open-vsx/publish-extensions#228]
The [erlang-ls extension](https://open-vsx.org/extension/erlang-ls/erlang-ls) is missing the compiled erlang language server.

When running it, there is an output error:
```
Failed to open file: /home/bryphe/.config/oni2/extensions/erlang-ls.erlang-ls-0.0.15/erlang_ls/_build/debug/bin/erlang_ls\n
```

It looks like the steps here: https://github.com/open-vsx/publish-extensions/blob/3a119cb944a2e8c2d45127233be70bce7420b3b3/extensions.json#L361

aren't including the build steps necessary to bring in the language server:
https://github.com/erlang-ls/vscode/blob/79b7b5b8a180f00c640d9aa772e2fc4d9d7180e7/package.json#L23

It looks like there are a few things that would need to be fixed:
1) Run `git submodule update --init` on the repo
2) Ensure the `erlang` and `rebar3` dependencies are available
3) Run the `npm compile` build step

**Comment by jankeromnes** [comment]
Hi @bryphe, good catch, many thanks! 🙏 

Please note that https://github.com/open-vsx/publish-extensions/pull/217 is already adding `npm run compile` as an extra `"prepublish"` step for this extension, which should hopefully fix this problem.

After it's merged, we should remove the broken extension from OpenVSX, so that a fixed binary can be re-uploaded.

**Comment by bryphe** [comment]
Oh, that's great, that's awesome that it's already being addressed - thanks @jankeromnes !

**Comment by jankeromnes** [comment]
The fix has been merged (thanks again @chintan-mishra!).

@spoenemann could you please delete https://open-vsx.org/extension/erlang-ls/erlang-ls so that it can be re-published? (`0.0.15` is the only published version, but it's broken.)

**Comment by spoenemann** [comment]
Done.

**Comment by jankeromnes** [comment]
Awesome, thanks @spoenemann!

The extension has since been re-published: https://open-vsx.org/extension/erlang-ls/erlang-ls

@bryphe / @chintan-mishra could you please double-check that it works as expected now? 🙏

**Comment by chintan-mishra** [comment]
The extension loads the Erlang Language Server(LS) nicely either by installing from VSCodium marketplace or downloading the `*.vsix` from `OpenVSX` and installing it manually.

I have yet to do a complete testing but I believe everything should work just fine.

**Note**: This is only applicable to those who had earlier installed the broken plugin.
**TL;DR**: Delete cached extensions
You can safely delete the folder `$HOME/.config/VSCodium/CachedExtensionVSIXs` and install the plugin. If you do not do this then the VSCodium will use the cached version of the plugin and you will be puzzled with the errors in the output.

**Comment by bryphe** [comment]
I tried the latest extension in Onivim, and can confirm it works now with the update. Thanks @chintan-mishra @jankeromnes @spoenemann !
